File: Cenfomon/Cenfomon_COI/bl/entities/NPC.ts
import { Dialog } from "./Dialog";


export class NPC{
    private _id:number;
    private _dialog:Array<Dialog>;
    private _picture:string;


    constructor(id: number, dialog: Array<Dialog>, picture: string) {
        this._id = id;
        this._dialog = dialog;
        this._picture = picture;
    }


    getid(): number {
        return this._id;
    }

    setid(value: number): void {
        this._id = value;
    }

    get dialog(): Array<Dialog> {
        return this._dialog;
    }

    set dialog(value: Array<Dialog>) {
        this._dialog = value;
    }

    get picture(): string {
        return this._picture;
    }

    set picture(value: string) {
        this._picture = value;
    }
}

File: Cenfomon/Capa_Logica/src/main/java/com/cenfomon/Cenfomon_Capa_Logica/NPC.java
package com.cenfomon.Cenfomon_Capa_Logica;

import java.util.ArrayList;

public class NPC {
    public int id;
    public ArrayList<Dialog> dialogs;
    public String picture;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public ArrayList<Dialog> getDialogs() {
        return dialogs;
    }

    public void setDialogs(ArrayList<Dialog> dialogs) {
        this.dialogs = dialogs;
    }

    public String getPicture() {
        return picture;
    }

    public void setPicture(String picture) {
        this.picture = picture;
    }
}
